[2022-02-21 16:53:05,661] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: test_path.get_path scheduled__2022-01-04T00:00:00+00:00 [queued]>
[2022-02-21 16:53:05,672] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: test_path.get_path scheduled__2022-01-04T00:00:00+00:00 [queued]>
[2022-02-21 16:53:05,672] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-21 16:53:05,672] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-02-21 16:53:05,673] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-21 16:53:05,691] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): get_path> on 2022-01-04 00:00:00+00:00
[2022-02-21 16:53:05,698] {standard_task_runner.py:52} INFO - Started process 64 to run task
[2022-02-21 16:53:05,702] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'test_path', 'get_path', 'scheduled__2022-01-04T00:00:00+00:00', '--job-id', '6', '--raw', '--subdir', 'DAGS_FOLDER/test.py', '--cfg-path', '/tmp/tmpa42yivzs', '--error-file', '/tmp/tmpeu7ix0j0']
[2022-02-21 16:53:05,702] {standard_task_runner.py:77} INFO - Job 6: Subtask get_path
[2022-02-21 16:53:05,826] {logging_mixin.py:109} INFO - Running <TaskInstance: test_path.get_path scheduled__2022-01-04T00:00:00+00:00 [running]> on host 7c24e16e6467
[2022-02-21 16:53:05,891] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=test_path
AIRFLOW_CTX_TASK_ID=get_path
AIRFLOW_CTX_EXECUTION_DATE=2022-01-04T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-01-04T00:00:00+00:00
[2022-02-21 16:53:05,891] {logging_mixin.py:109} INFO - /opt/***
[2022-02-21 16:53:05,891] {python.py:175} INFO - Done. Returned value was: None
[2022-02-21 16:53:05,905] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=test_path, task_id=get_path, execution_date=20220104T000000, start_date=20220221T165305, end_date=20220221T165305
[2022-02-21 16:53:05,954] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-21 16:53:05,983] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-02-21 17:01:48,050] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: test_path.get_path scheduled__2022-01-04T00:00:00+00:00 [queued]>
[2022-02-21 17:01:48,062] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: test_path.get_path scheduled__2022-01-04T00:00:00+00:00 [queued]>
[2022-02-21 17:01:48,062] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-21 17:01:48,062] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-02-21 17:01:48,062] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-21 17:01:48,076] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): get_path> on 2022-01-04 00:00:00+00:00
[2022-02-21 17:01:48,082] {standard_task_runner.py:52} INFO - Started process 64 to run task
[2022-02-21 17:01:48,085] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'test_path', 'get_path', 'scheduled__2022-01-04T00:00:00+00:00', '--job-id', '5', '--raw', '--subdir', 'DAGS_FOLDER/test.py', '--cfg-path', '/tmp/tmpfar_mmty', '--error-file', '/tmp/tmpit0ar9xy']
[2022-02-21 17:01:48,085] {standard_task_runner.py:77} INFO - Job 5: Subtask get_path
[2022-02-21 17:01:48,148] {logging_mixin.py:109} INFO - Running <TaskInstance: test_path.get_path scheduled__2022-01-04T00:00:00+00:00 [running]> on host ce809d68944a
[2022-02-21 17:01:48,203] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=test_path
AIRFLOW_CTX_TASK_ID=get_path
AIRFLOW_CTX_EXECUTION_DATE=2022-01-04T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-01-04T00:00:00+00:00
[2022-02-21 17:01:48,203] {python.py:175} INFO - Done. Returned value was: /nba_database
[2022-02-21 17:01:48,222] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=test_path, task_id=get_path, execution_date=20220104T000000, start_date=20220221T170148, end_date=20220221T170148
[2022-02-21 17:01:48,259] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-21 17:01:48,286] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-02-21 19:10:32,978] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: test_path.get_path scheduled__2022-01-04T00:00:00+00:00 [queued]>
[2022-02-21 19:10:32,990] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: test_path.get_path scheduled__2022-01-04T00:00:00+00:00 [queued]>
[2022-02-21 19:10:32,990] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-21 19:10:32,990] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-02-21 19:10:32,990] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-21 19:10:32,999] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): get_path> on 2022-01-04 00:00:00+00:00
[2022-02-21 19:10:33,003] {standard_task_runner.py:52} INFO - Started process 64 to run task
[2022-02-21 19:10:33,006] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'test_path', 'get_path', 'scheduled__2022-01-04T00:00:00+00:00', '--job-id', '5', '--raw', '--subdir', 'DAGS_FOLDER/test.py', '--cfg-path', '/tmp/tmpu2xpmr3h', '--error-file', '/tmp/tmp57qo9c77']
[2022-02-21 19:10:33,007] {standard_task_runner.py:77} INFO - Job 5: Subtask get_path
[2022-02-21 19:10:33,050] {logging_mixin.py:109} INFO - Running <TaskInstance: test_path.get_path scheduled__2022-01-04T00:00:00+00:00 [running]> on host 1594888d6c38
[2022-02-21 19:10:33,084] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=test_path
AIRFLOW_CTX_TASK_ID=get_path
AIRFLOW_CTX_EXECUTION_DATE=2022-01-04T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-01-04T00:00:00+00:00
[2022-02-21 19:10:33,085] {python.py:175} INFO - Done. Returned value was: ['/home/***/.local/bin', '/usr/local/lib/python37.zip', '/usr/local/lib/python3.7', '/usr/local/lib/python3.7/lib-dynload', '/home/***/.local/lib/python3.7/site-packages', '/usr/local/lib/python3.7/site-packages', '/opt/***/dags', '/opt/***/config', '/opt/***/plugins']
[2022-02-21 19:10:33,099] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=test_path, task_id=get_path, execution_date=20220104T000000, start_date=20220221T191032, end_date=20220221T191033
[2022-02-21 19:10:33,140] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-21 19:10:33,156] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
